backend/settings_base.py

```diff
@@ -35,7 +35,7 @@
     'news.apps.NewsConfig',
     'messmenu.apps.MessmenuConfig',
     'other.apps.OtherConfig',
-
+    'venter',
```

Comment:
`venter.apps.VenterConfig` instead of just `venter`. This is the new convention.